### ISSUE TITLE
More diversed lunch break

### ISSUE THREAD

**Issue by DaEgi01** [PropaneDragon/RushHour#54]
I have an issue with the lunch break habits of my cims.
They tend to be crazy about certain commercial buildings, while completely ignoring other opportunities. Problem is most visible for universities and high-schools, where during lunch time a giant crowd will rush out, all going to the same store.
I guess it is not due to a new iPhone release in that store or something ;)
It looks funny and hilarious.
What is the default AI behaviour here?
Could you make it more diverse without affecting game performance too much?

**Comment by PropaneDragon** [comment]
Yeah, this is due to how the AI get a random building. They all pick the same building because it's generated from their current building I believe. I might be able to do something in the future but I'm not sure.
